packages/web_drivers/lib/chrome_driver_installer.dart

```diff
@@ -30,10 +30,17 @@ class ChromeDriverInstaller {
   String get downloadUrl =>
       '$chromeDriverUrl$chromeDriverVersion/${driverName()}';
 
-  bool get isInstalled =>
-      io.File(path.join(driverDir.path, 'chromedriver')).existsSync();
+  io.File get installation =>
+      io.File(path.join(driverDir.path, 'chromedriver'));
 
-  Future<void> start() async {
+  bool get isInstalled => installation.existsSync();
+
+  ChromeDriverInstaller() : this.chromeDriverVersion = '';
+
+  ChromeDriverInstaller.withVersion(String version)
+      : this.chromeDriverVersion = version;
+
+  Future<void> start({bool alwaysInstall = false}) async {
     // Install Chrome Driver.
     try {
       final bool isSuccessfullyInstalled = await install();
```

Comment:
Did you mean to pass `alwaysInstall` to `install()`?

Reply:
yes! sorry.